### DISCUSSION TITLE
Courseplay kann auf der HB V1.3 im Weinberg keine Feldgrenzen verwerten!?

### DISCUSSION THREAD

**Discussion by BrunoMoser13** [Courseplay/Courseplay_FS22#2928] [Q&A]
Ich habe auf der HofBergmann mit der letzte Version 1.3 das Problem, dass ich im Weinberg kein CP benutzen kann, da die Felder nicht automatisch erkannt werden und wenn ich die Feldgrenzen manuell einstelle diese nicht berücksichtig werden. 
Wenn ich eine Job erstellen will und die Feldgrenze wie auf dem Bild zu sehen ist innerhalb von CP-6 setze wird eine Diagonale als Feldgrenze angezeigt und nicht meine manuell erstellte. Dadurch kann ich keinen Kurs erstellen der meine CP-6 abdeckt. Es wird ein Kurs erstellt der die diagonale Grenze benutzt. Bei der alten Version der HB bis 1.2 hat es immer funktioniert.

Die Ersteller der HB haben mir erklärt, dass der Weinberg aufgrund der Neigung der Reben manuell auf GPS Mod und VCA optimiert wurde und sich CP in der aktuellen Version nicht "auskennt"

Gibt es ein update von CP für dieses Problem?

link zum Post auf LSFM [](https://www.lsfarming-mods.com/wbb/thread/8910-courseplay-im-weinbau/?postID=83582#post83584)
[https://www.lsfarming-mods.com/wbb/thread/8910-courseplay-im-weinbau/?postID=83582#post83584](url)

![Problem mit Feldgrenze_1](https://github.com/Courseplay/Courseplay_FS22/assets/62007756/b3c2eeb1-4b23-4ec1-84a3-fa0df3ea6402)

**Comment by Tensuko** [comment]
> Gibt es ein update von CP für dieses Problem?

Nein, gibt es nicht.

https://github.com/Courseplay/Courseplay_FS22/issues/1683#issuecomment-1147388739
